include/nng/mqtt/mqtt_quic.h

```diff
@@ -17,7 +17,7 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
-
+#if defined(SUPP_QUIC)
```

Comment:
Actually, this file would not be included if NNG_PROTO_MQTT_QUIC_CLIENT option is not ON

Reply:
isnt this what we want?